lnrpc/websocket_proxy.go

```diff
@@ -50,15 +51,26 @@ var (
 	defaultProtocolsToAllow = map[string]bool{
 		"Grpc-Metadata-Macaroon": true,
 	}
+
+	// DefaultPingInterval is the default number of seconds to wait between
+	// sending ping requests.
+	DefaultPingInterval = time.Second * 30
+
+	// DefaultPongWait is the maximum duration we wait for a pong response
+	// to a ping we sent before we assume the connection died.
+	DefaultPongWait = time.Second * 5
```

Comment:
could we be less aggressive here? Is there any downside to increasing this? (im thinking 5s could be too little for slow connections, tor etc)

Reply:
I think this should be fine for most situations, as even with Tor 5 seconds is quite a long time. But I went ahead and made both numbers configurable, just in case.